guardian/admin.py

```diff
@@ -178,8 +178,8 @@ def obj_perms_manage_view(self, request, object_pk):
         )
 
         if request.method == 'POST' and 'submit_manage_user' in request.POST:
-            user_form = UserManage(request.POST)
-            group_form = GroupManage()
+            user_form = self.get_obj_perms_user_select_form(request)(request.POST)
+            group_form = self.get_obj_perms_group_select_form(request)(request.POST)
```

Comment:
Here you modified the ``group_form`` constructor call to pass the ``request.POST`` parameter - was this intended?